python/Ganga/Core/MonitoringComponent/Local_GangaMC_Service.py

```diff
@@ -81,16 +86,16 @@ def _execUpdateAction(self):
             if self.should_stop():
                 break
 
-            tpFreeThreads -= 1
             #setattr(threading.currentThread(), 'action', action)
             log.debug("Qin's size is currently: %d" % Qin.qsize())
             log.debug("%s running..." % threading.currentThread())
-
+            self._alive = True
```

Comment:
What I mean is: what does `self._alive` mean? Is it related to the standard Python `threading` concept of `is_alive()` or is it a different thing? If it's different then I think that calling it something which makes this clear is a better idea.